src/pages/profile.tsx

```diff
@@ -70,22 +73,19 @@ const profileStats = (
   );
   const winRate = MatchStatistics.playerWinRate(playerMatches, userId);
   const { highestLoseStreak, highestWinStreak } =
-    MatchStatistics.getPlayersStreak(matchesWithPlayers, userId);
+    MatchStatistics.getPlayersStreak(matches, userId);
```

Comment:
I guess `playerMatches` could be used here instead of `matches`, just for less computation.

Reply:
done